documentation/design_docs/master_variables.csv

```diff
@@ -1,16 +1,16 @@
-name,description,format,bounds,default,origin,use,current,list
-tpt_allow_deesc,allow any de-esc (Korn et al.) for 3+3,binary,"T,F",F,spec_three,main_ui,y,n
+﻿name,description,format,bounds,default,origin,use,current,list
+tpt_allow_deesc,allow any de-esc (Korn et al.) for 3+3,binary,"T,F",F,spec_tpt,main_ui,y,n
 skip_esc,allow skipping doses while escalating,binary,"T,F",F,spec_crm,main_ui,y,n
 skip_deesc,allow skipping doses while de-escalating,binary,"T,F",T,spec_crm,main_ui,y,n
 no_esc_if_observed_gt_target,no dose escalation if the overall observed DLT rate at the current dose level is above the target DLT rate,binary,"T,F",T,spec_crm,main_ui,y,n
 n_doses,number of dose levels,number,1:,5,non_spec,main_ui,y,n
 ttl,target toxicity level,number,0:1,0.2,non_spec,main_ui,y,n
 max_n,maximum sample size (always stop after n patients),number,0:,24,non_spec,main_ui,y,n
 start_dose,starting dose level,number,1:,1,non_spec,main_ui,y,n
-prior_var,prior estimate of variance,number,0:,0.5,spec_crm,main_ui,y,n
+prior_var,prior estimate of variance,number,0:,0.5,spec_crm,main_ui,y,y

```

Comment:
LISTS - any model specific parameters which are used in multiple designs are now recorded in lists. This is great, but we need to change the value recorded in the 'origin' column. Here it is listed as originating from 'spec_crm', meaning the CRM specific questionnaire. This may no longer be the case if other trial designs also record a prior variance. I think, then, that we need a new origin variable, maybe just 'spec' if its recorded in design specific questionnaires, or something else suitable. Happy to discuss further.